tests/conftest.py

```diff
@@ -26,11 +26,24 @@ def allowed_audiences() -> List[str]:
 
 
 @pytest.fixture(autouse=True)
-def clear_authz_collector():
+def clear_authz_collector() -> Generator:
```

Comment:
It would be better to be more precise: `Generator[None, None, None]`.

Python allows using `Iterator[None]` if you want to get rid of multiple `None` 😉